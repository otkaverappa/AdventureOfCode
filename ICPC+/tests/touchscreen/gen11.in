19
khtbavqtccm 10
dcpwdesrpuk
onxyyumgjyn
czqzqjcsvfw
mebqwtgtfwg
evgzjnkhami
qnycptzxrmd
tczmiuthbyc
hznwjwxvepi
fraarzrdcmh
dasyuzbsdkt
shcogzuvjzp 10
lbiktnofusg
oscnvwsfpko
tyustsbuhnw
rzreowaiera
ievezcvjrok
liffkzozxfy
olonmwtfwcl
tikwgunugad
lgpmuqkcqan
fpbbjganstx
crdygagjfrp 10
uglmspcsdhk
gltnlihgfly
jkejsnokeks
sykhaemkotv
jigrpmxamim
rtfghpntjft
totzbfqwaze
ittxstjagct
ljccyxlfqgt
lhwrxsrzyas
rzknikvlfgw 10
hkuexzwfuhm
tyklslmuddt
pqbblogtkpf
gvchrhcyuvz
girtulwaens
hogwwzilhpg
jwztyxpvyxe
pqacoaibtsr
zopawchnktk
jokidihaoys
odgrnjkfclv 10
rnutxhdqtoe
ewjoflcgdqj
lhwutewgzpu
tkrdnjwdpcb
yqhlspolabs
yvoctucioxs
hahqzmhbnfu
wsmlgxlhsme
iozcsjtsbas
kttwaxmiria
fwjdjsxvzfm 10
yjeicwlfryy
mwajikatlgr
wlaqkyprkqa
qyfplkgjkug
meqxejkmcgz
dwjdocpedfe
kupwczixhud
xrjjbwljxqg
jvwlkaqseao
twrugqdbvdu
emwczhaspjn 10
luylnqrnenm
xjspnvkqqre
otdvtvlelyb
jkobddhqrfa
kwnhhfxyjnt
olpmwtxuwgg
khkqqchvcts
rbbwabhowwz
lixhftdncqw
mgmopktjcmm
fjmiqceobqy 10
axducqwsoka
baqltbpgnwr
zgjckafcygb
cceuyylkbmk
rzguqohmhis
qmequhpakte
qperaosoaln
gfdwmrdwjwj
qodzffjyjcq
ntqdlfdwumd
aiprondkwty 10
bsegderfugy
njlupjobmol
dizrljnekry
qxburjrxhej
vayeobrfmun
olazbejuezr
hwkqniatrvt
rzjssqgndwa
dwdjhxnjrxf
bnulnpekixk
trekytpuuts 10
zeazulsrqve
mjubnellqec
xqcqgzmzrof
tnzehswczkl
vnbzymsfqpv
thbuucnikhx
lneekgduuyi
vzijtpzknus
oqotewdlvqy
zukhzhbzryb
bhurjhedbuu 10
ppanulklljh
yshhugyshbc
eslnyqovmfk
muhxeslpzlh
gsbpswytaeo
lrmbhhoouai
bamvnduylds
oukmufmztxs
ibappolrzot
njireicjaqf
mfzsrynrrvs 10
tljjycinwww
encnrnpjsco
swisjzjfres
cosewfauehh
gwawlmopail
ibxjmearycf
qjbwlwasfiq
igcuvtufhfg
eotkrkivpye
suqqwkyhbgn
dakywqfdyju 10
rtndbiucnpw
ffvpfesnrxp
ewowbrwlnog
atkjomyfkfm
hwslpffmxji
lhkedxstfup
pfddfkqkwxh
ojywoktzvgg
fklecfjxwbc
zgilwuhudyd
bwtorsjxaqh 10
nwmuflqinrp
xclukhzikbg
frxzcuzubnq
nhvyyeopvnu
gjeojoamuid
rhhnjbqwrgf
oefsuxpkrwv
xmgloshwxae
enphdlyjsnq
ahmxxyqvunh
ayxvfusfazu 10
pgxchjwwzwd
ntcmjxiwgkw
efdaakaberh
evqnrmooqbj
snvsxtyiscn
vcpieqmxass
qhkdvavzjpm
ghmciuumjrp
zzvrmtrglko
xplxmkjbzpl
lrvfnhqfype 10
ujspazampyb
axnlgrkyevp
zcdgskhkobt
hqtisgxsjxs
xiyquwupovt
tbmdiwrlpad
kiysfscdkbn
jrihnzvitql
ftwdnhsnney
nxdfbiqeybv
iikhgscwdhs 10
zmhjewylulk
aqlkipijmsu
wacqcavmsvy
bgfygssrcuj
nfsepdqizok
deoezcywdae
kykcsdenntu
hxlmptoffrl
uxmwwlbwplu
cnohtbwmxfl
jsceihkbugz 10
hevsgrjrnlh
bvafrpexcnr
fuzogajobss
wkaptuehbic
dntstsxjjed
kslldzovrlh
uabohkrpoue
hpxcmglqlwk
wjqvxlpktjn
wxuiqjwknfb
khnttfersdc 10
iyzvpmqyzmx
tunesarybdf
pwawdtogvwg
vuwhkuixudt
jhmjamldtac
uyhnpnjlvgf
rpsoxpiakrj
ybjkomeoqao
xpdkyrggyzv
qnuhxwsrfsu
