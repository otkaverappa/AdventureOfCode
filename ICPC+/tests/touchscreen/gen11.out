mebqwtgtfwg 32
fraarzrdcmh 36
evgzjnkhami 42
onxyyumgjyn 42
czqzqjcsvfw 43
dcpwdesrpuk 44
qnycptzxrmd 44
hznwjwxvepi 46
tczmiuthbyc 46
dasyuzbsdkt 48
lbiktnofusg 36
liffkzozxfy 40
oscnvwsfpko 42
lgpmuqkcqan 43
olonmwtfwcl 43
tyustsbuhnw 45
tikwgunugad 47
ievezcvjrok 51
fpbbjganstx 52
rzreowaiera 59
rtfghpntjft 32
sykhaemkotv 36
gltnlihgfly 39
ljccyxlfqgt 39
ittxstjagct 42
jkejsnokeks 44
lhwrxsrzyas 44
uglmspcsdhk 45
jigrpmxamim 49
totzbfqwaze 49
gvchrhcyuvz 32
tyklslmuddt 32
pqbblogtkpf 37
girtulwaens 38
pqacoaibtsr 44
jokidihaoys 45
jwztyxpvyxe 49
hogwwzilhpg 54
hkuexzwfuhm 55
zopawchnktk 55
hahqzmhbnfu 33
rnutxhdqtoe 35
kttwaxmiria 36
lhwutewgzpu 36
wsmlgxlhsme 36
tkrdnjwdpcb 38
iozcsjtsbas 40
yqhlspolabs 40
ewjoflcgdqj 41
yvoctucioxs 47
twrugqdbvdu 24
dwjdocpedfe 28
xrjjbwljxqg 33
jvwlkaqseao 38
mwajikatlgr 42
meqxejkmcgz 43
yjeicwlfryy 46
kupwczixhud 47
qyfplkgjkug 50
wlaqkyprkqa 57
otdvtvlelyb 37
jkobddhqrfa 44
olpmwtxuwgg 46
rbbwabhowwz 46
kwnhhfxyjnt 47
xjspnvkqqre 49
khkqqchvcts 50
luylnqrnenm 50
lixhftdncqw 51
mgmopktjcmm 58
baqltbpgnwr 40
gfdwmrdwjwj 40
rzguqohmhis 42
cceuyylkbmk 44
qperaosoaln 45
zgjckafcygb 46
qodzffjyjcq 47
axducqwsoka 52
ntqdlfdwumd 55
qmequhpakte 55
dizrljnekry 34
vayeobrfmun 35
hwkqniatrvt 38
njlupjobmol 42
dwdjhxnjrxf 43
bnulnpekixk 44
qxburjrxhej 44
bsegderfugy 45
rzjssqgndwa 51
olazbejuezr 52
thbuucnikhx 26
lneekgduuyi 35
mjubnellqec 35
vzijtpzknus 38
zeazulsrqve 47
oqotewdlvqy 48
xqcqgzmzrof 48
zukhzhbzryb 49
vnbzymsfqpv 50
tnzehswczkl 52
yshhugyshbc 30
njireicjaqf 32
bamvnduylds 38
eslnyqovmfk 39
lrmbhhoouai 39
gsbpswytaeo 42
oukmufmztxs 42
muhxeslpzlh 43
ibappolrzot 45
ppanulklljh 47
igcuvtufhfg 26
ibxjmearycf 30
suqqwkyhbgn 34
swisjzjfres 35
cosewfauehh 36
encnrnpjsco 40
tljjycinwww 43
qjbwlwasfiq 44
eotkrkivpye 45
gwawlmopail 45
lhkedxstfup 31
ewowbrwlnog 34
ffvpfesnrxp 35
atkjomyfkfm 36
zgilwuhudyd 36
fklecfjxwbc 37
hwslpffmxji 39
rtndbiucnpw 42
ojywoktzvgg 49
pfddfkqkwxh 52
xmgloshwxae 27
oefsuxpkrwv 40
nhvyyeopvnu 42
rhhnjbqwrgf 43
nwmuflqinrp 44
gjeojoamuid 49
enphdlyjsnq 50
xclukhzikbg 51
ahmxxyqvunh 55
frxzcuzubnq 60
pgxchjwwzwd 29
snvsxtyiscn 29
efdaakaberh 30
evqnrmooqbj 35
ghmciuumjrp 40
zzvrmtrglko 40
xplxmkjbzpl 44
qhkdvavzjpm 45
ntcmjxiwgkw 48
vcpieqmxass 50
nxdfbiqeybv 26
ftwdnhsnney 28
kiysfscdkbn 38
hqtisgxsjxs 39
jrihnzvitql 43
tbmdiwrlpad 44
zcdgskhkobt 44
ujspazampyb 46
axnlgrkyevp 53
xiyquwupovt 54
bgfygssrcuj 26
kykcsdenntu 31
deoezcywdae 36
cnohtbwmxfl 39
hxlmptoffrl 46
uxmwwlbwplu 46
zmhjewylulk 46
wacqcavmsvy 47
nfsepdqizok 49
aqlkipijmsu 59
hevsgrjrnlh 30
uabohkrpoue 36
dntstsxjjed 39
hpxcmglqlwk 39
bvafrpexcnr 42
fuzogajobss 42
kslldzovrlh 43
wjqvxlpktjn 45
wkaptuehbic 45
wxuiqjwknfb 45
tunesarybdf 24
jhmjamldtac 29
iyzvpmqyzmx 31
qnuhxwsrfsu 33
uyhnpnjlvgf 33
xpdkyrggyzv 36
ybjkomeoqao 36
pwawdtogvwg 38
vuwhkuixudt 42
rpsoxpiakrj 53
